examples/basic.rs

```diff
@@ -14,6 +14,8 @@ async fn main() -> Result<()> {
 
     session.query("CREATE KEYSPACE IF NOT EXISTS ks WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1}", &[]).await?;
 
+    println!("{:#?}", session);
```

Comment:
I'm not sure about printing the whole session in the basic example.

`basic.rs` is the first example that someone learning how our driver works will run.
They might be startled that 2/3 of output is `Session` debug with all of its internals shown.

It's useful for verifying that the `Debug` impl really works and seeing what it looks like, but putting it in `basic.rs` is pretty bold. I think it would better to create a separate example which shows that it's possible to print session internals.

Reply:
Yes, the change in `basic.rs` was intended for my testing only, thank you for pointing this out.